src/test/java/com/es/phoneshop/model/product/service/CartServiceImplTest.java

```diff
@@ -58,7 +57,7 @@ public void givenHttpSession_WhenGetCart_ThenReturnNewCart() {
         verify(httpSession).getAttribute(anyString());
         verify(httpSession).setAttribute(anyString(), any());
         assertNotNull(result);
-        assertNotEquals(cart, result);
+        assertFalse(cart == result);
```

Comment:
Why not equals?

Reply:
Because they are equals. Last time, i think, i do not write equals in cart.
May be is it good write just equals?